### PR TITLE
current state of "walleyeDecline" figure, and text additions

### PR DESCRIPTION
**Pull request by jordansread**
animates the lines and makes the fish populations move too. 

image

### CODE REVIEW

**Comment by jordansread** [comment]
@lindsaycarr can you review and merge?

File: layout/templates/lakeWarming.mustache [hunk 1]
```diff
@@ -5,12 +5,8 @@
 </div>
 
 <div id="lakeWarming" class="center">
-	  <div class="list">
-	    <li>{{lakeWarming-l1}}</li>
-	    <li>{{lakeWarming-l2}}</li>
-	    <li>{{lakeWarming-l3}}</li>
-	    <li>{{lakeWarming-l4}}</li>
-	    <li>{{lakeWarming-l5}}</li>
-	  </div>
```
Comment (jordansread) [inline layout/templates/lakeWarming.mustache:14]: @jiwalker-usgs is this preferred?

File: layout/templates/walleyeDecline.mustache [hunk 2]
```diff
@@ -8,14 +8,10 @@
...
-		  <li>{{ walleyeDecline-l1 }}</li>
-		  <li>{{ walleyeDecline-l2 }}</li>
-		  <li>{{ walleyeDecline-l3 }}</li>
-		  <li>{{ walleyeDecline-l4 }}</li>
-		  <li>{{ walleyeDecline-l5 }}</li>
-		</ul>
+	  {{#walleyeDecline-bullets}}  <li>{{.}}</li>
+	  {{/walleyeDecline-bullets}}</ul>
 		
-	  <object id="walleyeDeclineSVG" type="image/svg+xml" class="svgFig" data="{{ wallyTrendsFig }}">
```
Comment (jordansread) [inline layout/templates/walleyeDecline.mustache:18]: remove class so this stretches to full width. Probably will want a different class for this one in the future

**Comment by lindsayplatt** [comment]
@jread-usgs I have been following along with your commits, but I don't think I'm the best person to review because you've really only been changing svg/xml stuff and the mustache files.

**Comment by jordansread** [comment]
@jiwalker-usgs can you review?

File: layout/templates/lakeWarming.mustache (hunk 1 above)
Comment (jiwalker-usgs) [inline layout/templates/lakeWarming.mustache:14]: yeah, I like this better, though the `<li>` stuff should be wrapped in `<ul>` or `<ol>`

Comment (jiwalker-usgs) [inline layout/templates/lakeWarming.mustache:14]: nevermind, missed it

**Comment by jordansread** [comment]
good to go @jiwalker-usgs ? keeping in mind I am trying to improve the viz code as we go to be less sandboxy.

**Comment by jiwalker-usgs** [comment]
Good to go :+1:

**Pull request merged**